style.css

```diff
@@ -1,54 +1,123 @@
 @import url("https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap");
 
-@media only screen and (max-width: 768px) and (min-width: 375px) {
+@media (min-width: 375px) and (max-width: 767px) {
   body {
+    max-width: 375px;
     margin: auto;
-    background-color: #fff;
   }
 
   .main {
-    width: 100vw;
-    height: 100vh;
+    width: 100%;
+    height: 100%;
+  }
+
+  .project-img > img {
+    max-width: 95%;
+    max-height: 100%;
+    display: block;
+    margin-left: 0.7rem;
+    margin-top: 1rem;
+    object-fit: cover;
+  }
+
+  .mylogo-mobile {
+    line-height: 80%;
+    width: 70%;
+    height: 2em;
+    font-size: 0.5em;
+  }
+
+  .contact-me {
+    height: 130vh;
+  }
+
+  .desktop {
+    display: none;
   }
 }
 
-@media (max-width: 1080px) {
-  body {
-    max-width: 750px;
+@media only screen and (min-width: 767px) {
+  * {
     margin: auto;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: "Poppins", sans-serif;
+    max-width: 767px;
   }
```

Comment:
[OPTIONAL] You have done massive work fixing all the issues that the previous reviewers pointed out 🔥. However,  there seems to be a little issue. I am only making this optional because the previous reviewers didn't mention this but I think it is really important that you fix this issue and I urge you to. On line 45, you have a hardcoded max-width value that is keeping the entire page fixed at that width (767 px). I think that this is not ideal for UI as it makes the whole page stay fixed in the middle on large screens. I think that removing this would be a very good idea so that you are able to position the elements correctly on the page without making it look shrinked to the middle.

![image](https://github.com/AliRezaBmeDu/Portfolio-Website/assets/107795045/ceb4ec75-9f51-400c-81f0-8fadba6191f3)